langlib/lang.function/src/main/java/org/ballerinalang/langlib/function/Call.java

```diff
@@ -64,7 +64,8 @@ public static Object call(BFunctionPointer<Object, Object> func, Object... args)
             throw ErrorCreator.createError(
                         getModulePrefixedReason(FUNCTION_LANG_LIB, INCOMPATIBLE_ARGUMENTS),
                         BLangExceptionHelper.getErrorDetails(RuntimeErrors.INCOMPATIBLE_ARGUMENTS,
-                        new BTupleType(paramTypes, restType, 0, false), new BTupleType(argTypes)));
+                        removeBracketsFromStringFormatOfTuple(new BTupleType(paramTypes, restType, 0, false)),
+                        removeBracketsFromStringFormatOfTuple(new BTupleType(argTypes))));
```

Comment:
This is wrong? Your error is

```
arguments of incompatible types: argument(s) of type(s) ''{0}'' cannot be passed to function expecting parameter(s) of type(s) ''{1}''
```

But you're passing param types first and argument types second?